### ISSUE TITLE
Authentication: Log in with Azure Active Directory with msal.js

### ISSUE THREAD

**Issue by tmszyman** [AzureAD/microsoft-authentication-library-for-js#119]
I created app on Application Registration Portal and connected it do Azure Authentication: Log in with Azure Active Directory. So my callback for app is "***.auth/login/aad/callback". How can I integrate this authentication with msal.js? I got error when I use acquireTokenSilent for generate token -
 "user_login_error". 

I also tried integrate app with https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/ and it works like:
1. Enter the web
2. Log in with Azure Active Directory
3. Open app
4. Again log in with loginRedirect
5. Enter redirectUri ("***.auth/login/aad/callback") and get response - IDX10500: Signature validation failed.

**Comment by rohitnarula7176** [comment]
@tmszyman You get this error when you try to acquire a token without a signed in user. Are you able to log in to your app? Please refer to the following sample on the repo https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/devApps/VanillaJSTestApp/index_LoginPopup.html on instructions on how to use the api's in msal.js.

**Comment by tmszyman** [comment]
Thanks for the answer. I looked at this sample. (https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/devApps/VanillaJSTestApp/index_LoginPopup.html).



Maybe I'll describe you how the app works in several situations.
I. Automatic login with Active Directory Authentication is **disabled**. Callback on Application Registration Portal is set to _home page of the app_. I used the code from https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/devApps/VanillaJSTestApp/index_LoginPopup.html
1. Enter the app without authorization
2. Click on button which runs loginPopup, after log in get token.

**Everything works, but I want authorization with Active Directory Authentication**



II. Automatic login with Active Directory Authentication is **enabled**. Callback on Application Registration Portal is set to _"***.auth/login/aad/callback"_. I used the code from https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/devApps/VanillaJSTestApp/index_LoginPopup.html
1. Enter the app with authorization and log in with Active Directory Authentication
2. acquireTokenSilent is sending error like "user_login_error".
3. Click on button which runs loginPopup, after log in I get error like "The reply address **** does not match the reply addresses configured for the application"


**To summarize, after login with Azure I get error from acquireTokenSilent "user_login_error".**


**III. I would like the app to behave as follows**:
1. Enter the app with authorization and log in with Active Directory Authentication
2. Get token

Can I do it like this?
